explorer/src/components/Gateways.ts

```diff
@@ -21,3 +28,21 @@ export function gatewayToGridRow(arrayOfGateways: GatewayResponse): GatewayRowTy
         host: gw.gateway.host || '',
       }));
 }
+
+export function gatewayEnrichedToGridRow(
+  gateway: GatewayResponseItem,
+  report: GatewayReportResponse,
+): GatewayEnrichedRowType {
+  return {
+    id: gateway.owner,
+    owner: gateway.owner,
+    identity_key: gateway.gateway.identity_key || '',
+    location: gateway?.gateway?.location || '',
+    bond: gateway.pledge_amount.amount || 0,
+    host: gateway.gateway.host || '',
+    clients_port: gateway.gateway.clients_port || 0,
+    mix_port: gateway.gateway.mix_port || 0,
+    routing_score: `${report.most_recent}%`,
+    avg_uptime: `${report.last_day || report.last_hour}%`,
+  };
```

Comment:
We should conform to the JS/TS naming convention whenever we can (ofc in JS/TS side), so when we are creating our types, use `camelCase` for field/variable declaration. Even if it maps to external data that uses its own different convention (`snake_case` in this case).

Reply:
@doums I was in doubt, as all fields on the explorer has this convention instead the camelCase. I'd say that here is an "explorer convention". What do you think, should i make here the exception using camelCase?